app/Dockerfile

```diff
@@ -2,11 +2,14 @@ FROM node
 
 ENV PORT 3000
 
-COPY . .
+RUN mkdir -p /app/startup_directory
```

Comment:
Siraj and I had changed this `Dockerfile` yesterday to fix the `next: command not found` issue. Does this issue persist with these changes? Or is that fixed as well?

Reply:
Should be fixed.